### PR TITLE
Update Russian and Ukrainian translations

### PR DESCRIPTION
**Pull request by Minoru**
It'd be nice if someone took a look before the weekend and made sure this has the right shape at least. I'll merge once I've got a review, or at the end of the week before cutting the release.

### CODE REVIEW

**Review by dennisschagt — APPROVED** [review]
LGTM

File: po/ru.po [hunk 1]
```diff
@@ -517,13 +519,13 @@ msgid "Exporting list of read articles..."
...
-#, fuzzy, c-format
+#, c-format
 msgid ""
 "%<PRIu64> unreachable feeds found. See `cleanup-on-quit` in newsboat(1) for "
 "details."
 msgstr ""
-"Найдены недостижимые ленты; используйте `cleanup-on-quit yes` или запустите "
-"`newsboat --cleanup`"
+"Найдено %<PRIu64> недостижимых лент; за подробностями см. `cleanup-on-quit` "
+"в newsboat(1)."
```
[the remark this answers is not in the document]
Comment (Minoru) [reply po/ru.po:528]: It's just a common shortening of a word "смотри", kind of like "Figure 3" might be shortened to "Fig. 3" in a textbook (written in English). Same with Ukrainian word "дивись".

Shortening also avoids the problem of how to address the reader. Russian and Ukrainian words change a bit depending on whether the style is formal or informal. Shortening the words erases the distinction, making the text slightly more neutral.

**Comment by Minoru** [comment]
Thanks! I'll merge this then.

File: po/ru.po (hunk 1 above)
[the remark this answers is not in the document]
Comment (dennisschagt) [reply po/ru.po:528]: Nice to know :) Thanks!

**Pull request merged**